arcaflow_plugin_pcp/pcp_schema.py

```diff
@@ -35,6 +35,17 @@ class PcpInputParams:
         schema.description(
             "Complete configuration file content for pmlogger as a multi-line string."
             " If no config file is provided, a default one will be generated."
+            " WARNING -- INPUT NOT VALIDATED - USE AT YOUR OWN RISK"
+        ),
+    ] = None
+    pmrep_conf: typing.Annotated[
+        typing.Optional[str],
+        schema.name("pmrep configuration file"),
+        schema.description(
+            "Complete configuration file content for pmrep as a multi-line string."
+            " If no config file is provided, a default one will be generated."
+            " This configuration is used internally for `pcp2json` and `pcp2csv`."
+            " WARNING -- INPUT NOT VALIDATED - USE AT YOUR OWN RISK"
```

Comment:
The warnings at lines 38 and 48 seem overblown, particularly the "USE AT YOUR OWN RISK" phrase.  (I mean, the whole software stack is pretty much offered under those terms....)

Moreover, I expect that the input _is_ validated by PCP...but a failure there could be inconvenient.

I suggest toning this down to "Note:  this input is not validated by the Arcaflow plugin prior to invoking PCP."  (Also, I think I would drop the word "internally".)

Reply:
This is important from an Arcaflow standpoint. We generally frown on `Any` types in the schema, and these inputs violate that standard. Not validating the input at the schema means that the validation error from the underlying tool will 1) happen at the end of the run, which could be hours later, and 2) could result in a non-user-friendly traceback from the calling commands. I think the warnings are important to align with our project philosophy.

Reply:
Those are excellent points, and the updated text is superb; however, I still don't think that solid-capitals are warranted.  Yes, it would be annoying -- possibly crushing -- to have a long run fail at the end due to un-checked inputs; but, for that to be really damaging means that the user took a great leap of faith, committing considerable resources, without so much as a simple preliminary test run...and, really, that wouldn't be our fault.